src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -200,26 +200,22 @@ public static void updateRepoConfig(RepoConfiguration config) {
 
         if (!Files.exists(configJsonPath)) {
             logger.info(String.format(MESSAGE_NO_STANDALONE_CONFIG, config.getLocation(), config.getBranch()));
-            return;
-        }
-
-        if (config.isStandaloneConfigIgnored()) {
+        } else if (config.isStandaloneConfigIgnored()) {
```

Comment:
No need to change the block format to `if ... elseif ... else ...`. Instead, we use original style to check if there are exceptional cases to deal with using `if` one by one.